src/Core/src/Platform/ImageSourcePartLoader.cs

```diff
@@ -23,48 +23,43 @@ namespace Microsoft.Maui.Platform
 {
 	public partial class ImageSourcePartLoader
 	{
+#if IOS || ANDROID || WINDOWS || TIZEN
 		IImageSourceServiceProvider? _imageSourceServiceProvider;
-		IImageSourceServiceProvider ImageSourceServiceProvider =>
-			_imageSourceServiceProvider ??= Handler.GetRequiredService<IImageSourceServiceProvider>();
+#endif
 
-		readonly Func<IImageSourcePart?> _imageSourcePart;
-		Action<PlatformImage?>? SetImage { get; }
-		PlatformView? PlatformView => Handler.PlatformView as PlatformView;
+		readonly WeakReference<ISetImageHandler> _handler;
 
 		internal ImageSourceServiceResultManager SourceManager { get; } = new ImageSourceServiceResultManager();
 
-		IElementHandler Handler { get; }
-
-		public ImageSourcePartLoader(
-			IElementHandler handler,
-			Func<IImageSourcePart?> imageSourcePart,
-			Action<PlatformImage?> setImage)
+		[Obsolete("To removed in a future release")]
```

Comment:
```suggestion
		[Obsolete("To be removed in a future release")]
```

Can you also surround the ctor in a `#if !NET9_0`

Reply:
A better one would be `#if !NET9_0_OR_GREATER` according to [this](https://github.com/dotnet/docs/blob/main/includes/preprocessor-symbols.md).

Reply:
@mattleibow is there an example somewhere of an API that will be remove in 9? I could proactively surround with `#if`, but maybe that's not actually what we'd want? You may not want to remove the APIs at the same time you move to a `net9.0` target framework.